content/about/dates.md

```diff
@@ -45,7 +45,7 @@ date_format = "02 Jan 2006"
   location = "Gdańsk, Poland"
   date_start = "2020-10-27"
   date_end = "2020-11-03"
-  description = """"""
+  description = "Summary: First meeting and first sprint ended successfully. "
```

Comment:
I think we can add here something about learning about object detectors, evaluating their results methods, annotations types. And, what is also important, get to know each other, get to know tools for comunication and planning